x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/action/TransportXPackUsageAction.java

```diff
@@ -59,7 +60,7 @@ protected XPackUsageResponse newResponse() {
     }
 
     @Override
-    protected void masterOperation(XPackUsageRequest request, ClusterState state, ActionListener<XPackUsageResponse> listener) {
+    protected void masterOperation(Task task, XPackUsageRequest request, ClusterState state, ActionListener<XPackUsageResponse> listener) {
```

Comment:
And here

Reply:
I don't think we can set it here. The same request is passed through executeLocally. The intention is to always do a local action, so it is really the same task, it just happens to be plugged in through other modules, instead of having the action to call directly.

Reply:
I will address this in a followup. executeLocally needs to allow a task to pass through and all its uses should be reviewed.